content/react/concepts/routing/routing.md

```diff
@@ -34,48 +34,34 @@ Below, a `BrowserRouter` is used to map the navigation `<Link/>` components to a
 ```jsx
 import React from 'react';
 
-import { BrowserRouter as Router, Switch, Route, Link } from 'react-router-dom';
+import { Routes, Route, Link } from 'react-router-dom';
```

Comment:
```suggestion
import {
  BrowserRouter as Router,
  Routes,
  Route,
  Link
} from 'react-router-dom';
```

According to the [new docs on React Router](https://reactrouter.com/docs/en/v6/getting-started/overview) we still need to use `<BrowserRouter>` so that `<Routes>` is being used in the correct context. Without it, I'm getting this error:

```
Error: useRoutes() may be used only in the context of a <Router> component.
```